### ISSUE TITLE
Building linux/aarch64 wheels requires qemu-user-static begin installed in the host container

### ISSUE THREAD

**Issue by dvandyk** [DavidMStraub/rundec-python#11]
@DavidMStraub I am sorry, my PR is likely going to trigger a timeout for the ongoing workflow (it will be hitting the 6h cancellation soon!) Options: you could revert my changes, or at least the commit for building for linux/aarch64.

I currently cannot see the log file of the action. I suspect that QEMU emulation makes it take quite a bit longer.

I was wondering if it would also be prudent to limit the builds for either all linux architectures or for linux/aarch64 to python 3.10 or higher? If you agree with this, I would also suggest to split the existing single job into multiple jobs, so that I can provide a different choice of ``env`` for each.

**Comment by DavidMStraub** [comment]
Hi,

doesn't look like a timeout issue:

```
  WARNING: The requested image's platform (linux/arm64/v8) does not match the detected host platform (linux/amd64) and no specific platform was requested
  9e6fcf5ba212fa8922e3c610e868705fa5d5523f728ca82074b47e6b3aadea3a
      + /bin/true
  exec /usr/local/bin/manylinux-entrypoint: exec format error
```

**Comment by dvandyk** [comment]
Fixed by 98f0a17f99ac48264d2d23bc5461b8b97fd0f65b
